pkg/operator/ceph/cluster/controller.go

```diff
@@ -173,6 +174,27 @@ func (c *ClusterController) StartWatch(namespace string, stopCh chan struct{}) {
 	} else {
 		logger.Infof("Disabling hotplug orchestration via %s", disableHotplugEnv)
 	}
+
+	// watch for "rook-ceph-operator-config" ConfigMap
+	configMapName := "rook-ceph-operator-config"
```

Comment:
Isn't `rook-ceph` hardcoded here?

Reply:
I don't understand the question.
I shifted this to a CONST in K8sutil since the name is fixed.